### PR TITLE
feat: Page break

### PR DESCRIPTION
**Pull request by areknawo**
This PR adds a page break block, dedicated for use with exporters to force page breaks in the documents.

- Page break block represented as a dashed line;
- Support for page break in PDF & DOCX exporters;

### CODE REVIEW

File: packages/xl-pdf-exporter/src/pdf/pdfExporter.test.tsx [hunk 1]
```diff
@@ -160,8 +160,8 @@ describe("exporter", () => {
     );
 
     const transformed = await exporter.toReactPDFDocument(testDocument);
-    const view = render(transformed);
-    const str = prettyDOM(view.container, undefined, { highlight: false });
+    const str = reactElementToJSXString(transformed);
```
Comment (YousefED) [inline packages/xl-pdf-exporter/src/pdf/pdfExporter.test.tsx:163]: what's the benefit of this library?

**Review by YousefED — CHANGES_REQUESTED** [review]
Nice!

Two things;
- I'm not sure we want it as part of the default schema. It's quite specific and only relevant if you're printing / exporting
- maybe add this CSS so it works for printing as well? https://www.w3schools.com/cssref/pr_print_pageba.php

File: packages/xl-pdf-exporter/src/pdf/pdfExporter.test.tsx (hunk 1 above)
Comment (areknawo) [reply packages/xl-pdf-exporter/src/pdf/pdfExporter.test.tsx:163]: Compared to `render` & `prettyDOM` it doesn't expect DOM/HTML-compatible JSX. Thus, it results in fewer warnings and fixes the `break` edge case I was facing (the `break` prop being removed from the snapshot due to incompatibility with DOM).

**Comment by areknawo** [comment]
- [x] Add CSS styles to support page break in printing;
- [ ] Remove page break from the default schema;

**Review by YousefED — COMMENTED** [review]
Couple of things:

- Can we align the drag handle better?

- can we make the design a bit nicer? e.g. dotted line that's grey instead of black dashes? Maybe @areknawo can later also add "Page break" in the middle, but that's non blocking
- The "Default schema" showcase actually doesn't show the default schema any more because it also adds multi-column + page break. We should update the example to reflect this
- Maybe add the page break block to the pdf and docx examples? Because now there's no easy way to test this functionality easily

**Review by YousefED — APPROVED** [review]

**Comment by YousefED** [comment]
closes #1179
